styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,12 +1,14 @@
 .footer-menu-link {
   font-size: 1.2rem;
   color: darken($light-gray, 15%);
+  text-decoration: none;
```

Comment:
Hey we forgot to add this to Roger's original footer PR.  Can you add it here? 
```
  &:hover, &:focus {
    color: $white;
  }
```

Otherwise the mouseover links are blue and that looks weird on the dark background.